deck.gl__core/index.d.ts

```diff
@@ -2716,8 +2742,11 @@ declare module "@deck.gl/core/controllers/orbit-controller" {
 	}
 }
 declare module "@deck.gl/core/views/orbit-view" {
-	import View from "@deck.gl/core/views/view";
+	import { View, ViewProps } from "@deck.gl/core/views/view";
```

Comment:
should this be `import View, { ViewProps } from "@deck.gl/core/views/view";` ?

Reply:
Yes, sorry! Fixed